gymnasium_robotics/envs/multiagent_mujoco/mujoco_multi.py

```diff
@@ -0,0 +1,574 @@
+import gymnasium
+import numpy
+import pettingzoo
+
+from .coupled_half_cheetah import CoupledHalfCheetah
```

Comment:
Could you use absolute imports not relative imports. Repeat for all imports

Reply:
Done (on a local repo, will be posted later)